backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/ShipmentScheduleRepository.java

```diff
@@ -227,5 +273,14 @@ private void save(@NonNull final ShipmentSchedule shipmentSchedule)
 
 		@Builder.Default
 		boolean includeProcessed = false;
+
+		@Builder.Default
+		boolean fromCompleteOrderOrNullOrder = false;
+
+		@Builder.Default
+		boolean orderByOrderId = false;
+		@Builder.Default
+		boolean noFutureExportOfOrder = false;
```

Comment:
i could understand `noFutureExportOfOrder` only after looking at the code in `ShipmentScheduleRepository`..
pls add a javadoc-comment to it, oooor think of a better name...like..maybe sth like `onlyIfAllFromOrderExportable`?

Reply:
Why not both? :)
Done